providers/hexonet/domains.go

```diff
@@ -21,6 +21,9 @@ func (n *HXClient) EnsureDomainExists(domain string) error {
 		}
 	} else if code == 531 {
 		return n.GetHXApiError("Not authorized to manage dnszone", domain, r)
+		// FIXME(tlim) go-staticcheck reports:
+		// identical expressions on the left and right side of the '||' operator (SA4000)
+		// Perhaps the right side should be n.IsError() or deleted?
 	} else if r.IsError() || r.IsError() {
```

Comment:
Morning @tlimoncelli 

I suggest to rewrite this to

```go
} else if r.IsError() || r.IsTmpError() {
```

Probably an oversight on my end some time ago.
Find tje Reference to `IsTmpError` function [here](https://github.com/hexonet/go-sdk/blob/master/responsetemplate/responsetemplate.go#L96).

Reply:
Changed as suggested. If it passed CI, I'll mark this as done.